src/System.Private.CoreLib/src/System/MulticastDelegate.cs

```diff
@@ -542,13 +543,13 @@ protected override MethodInfo GetMethodImpl()
                 if ((_methodBase == null) || !(_methodBase is MethodInfo))
                 {
                     IRuntimeMethodInfo method = FindMethodHandle();
-                    RuntimeType declaringType = RuntimeMethodHandle.GetDeclaringType(method);
+                    RuntimeType? declaringType = RuntimeMethodHandle.GetDeclaringType(method);
```

Comment:
When can `GetDeclaringType` return null?

Reply:
Ah, ok - it is another one of those assigned later cases. Note that the value assigned later to it cannot be null either. You can change `GetType() as RuntimeType` to `(RuntimeType)GetType()`.

Reply:
Changed, although changes like this are low risk on their own, we are changing so many files I am concerned we will introduce some subtle bug.

Reply:
Yep, based on my experience from past similar annotation exercises in the past (SAL in C/C++, ...), this will introduce some subtle bugs, no matter how hard you try. I think it is better to optimize for long term code readability and do local fixups up as necessary.